googlemock/include/gmock/gmock-matchers.h

```diff
@@ -4597,7 +4594,7 @@ UnorderedPointwise(const Tuple2Matcher& tuple2_matcher,
 
   // Create a matcher for each element in rhs_container.
   ::std::vector<internal::BoundSecondMatcher<Tuple2Matcher, Second> > matchers;
-  for (typename RhsStlContainer::const_iterator it = rhs_stl_container.begin();
+  for (auto it = rhs_stl_container.begin();
```

Comment:
Would a range-based `for` loop be OK here as well? (Just asking. I'm not a maintainer.)

Reply:
I think so!